test/ffmpeg-vaapi/util.py

```diff
@@ -109,3 +109,22 @@ def load_test_spec(*ctx):
   spec.update(get_media()._get_test_spec("ffmpeg-vaapi", *ctx))
 
   return spec
+
+@memoize
+def transcodeformat_hwhw_swhw(format):
+  return {
+    "avc"    : "h264_vaapi",
+    "hevc"   : "hevc_vaapi",
+    "mpeg2"  : "mpeg2_vaapi",
+    "mjpeg"  : "mjpeg_vaapi",
+  }.get(format, None)
+
+
+@memoize
+def transcodeformat_hwsw(format):
+  return {
+    "avc"    : "h264",
+    "hevc"   : "hevc",
+    "mpeg2"  : "mpeg2video",
+    "mjpeg"  : "mjpeg",
+  }.get(format, None)
```

Comment:
Perhaps these two methods can be combined somehow to simplify point of call.

Reply:
...better yet, just eliminate these methods altogether and pass the ffcodec from the test case as a variable

Reply:
removed the methods and passed codec format from test